AES.Entities/Tables/EducationHistory.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace AES.Entities.Tables
+{
+    public class EducationHistory
+    {
+        [Key]
+        [Required]
+        public int ID { get; set; }
+
+        [Required]
+        public ApplicantUser Applicant { get; set; }
+
+        [Required]
+        [StringLength(128)]
+        public string SchoolName { get; set; }
+
+        // Not required as the applicant may not know
+        [StringLength(128)]
+        public string SchoolAddress { get; set; }
+
+        [Required]
+        [StringLength(128)]
+        public string SchoolCity { get; set; }
+
+        // Not required as the applicant may not know or it may not be relevant
+        [StringLength(64)]
+        public string SchoolState { get; set; }
+
+        // Not required as the applicant may not know or it may not be relevant
+        [StringLength(16)]
+        public string SchoolZip { get; set; }
+
+        [Required]
+        [StringLength(128)]
+        public string SchoolCountry { get; set; }
+
+        // Not required as the applicant may not know
+        [StringLength(32)]
+        public string SchoolPhone { get; set; }
```

Comment:
Thinking 16 should be fine for this, assuming international numbers. But if you want, just drop international support.